app/assets/javascripts/admin/addon/components/plugin-commit-hash.hbs

```diff
@@ -0,0 +1,5 @@
+<a
+  href={{this.plugin.commit_url}}
+  class="current commit-hash"
+  title={{this.plugin.commit_hash}}
+>{{this.shortCommitHash}}</a>
```

Comment:
I still think my concern where `commit_hash` is `nil` needs to be addressed. When `commit_hash` is `nil`, we're generating a some what invalid link.

Reply:
I added an "if" condition. Now it won't display the invalid link if `commit_hash` is not available.